reciprocalspaceship/utils/grid.py

```diff
@@ -0,0 +1,55 @@
+import gemmi
+import numpy as np
+
+from reciprocalspaceship.decorators import cellify, spacegroupify
+
+
+@cellify
+@spacegroupify
+def get_gridsize(cell, dmin, sample_rate=3.0, spacegroup=None):
```

Comment:
I'd prefer `get_fourier_grid_size` or `get_reciprocal_grid_size`

Reply:
done -- for consistency elsewhere, I've made it `get_reciprocal_grid_size`